src/lib/userStorage/UserStorage.js

```diff
@@ -1,13 +1,16 @@
 //@flow
 import { default as goodWallet } from '../wallet/GoodWallet'
 import getDB from '../realmdb/RealmDB'
+import { UserProfileStorage } from './UserProfileStorage'
 import { UserStorage } from './UserStorageClass'
 import UserProperties from './UserProperties'
 
 const db = getDB()
 
 const userStorage = new UserStorage(goodWallet, db, new UserProperties(db))
+const storage = new UserProfileStorage(goodWallet, db)
```

Comment:
just like UserProperties is input to UserStorage so should be UserProfileStorage
everywhere in the wallet we just use UserStorage